src/main/java/uk/gov/hmcts/cmc/claimstore/config/JacksonConfiguration.java

```diff
@@ -16,6 +17,7 @@
     public ObjectMapper objectMapper() {
         return new ObjectMapper()
             .registerModule(new Jdk8Module())
+            .registerModule(new Jackson2HalModule())
```

Comment:
Is this needed? shouldn't the client be handling this?

Reply:
removed